docs/sources/tutorials/alerting-get-started/index.md

```diff
@@ -78,9 +122,8 @@ In this step, we'll set up a new [contact point](https://grafana.com/docs/grafan
 
 1. In your browser, **sign in** to your Grafana Cloud account.
 
-   {{< admonition type="note" >}}
-   **OSS users**: To log in, navigate to localhost:3000, where Grafana is running locally.
-   {{< /admonition >}}
+   **NOTE:**
+   **OSS and Killercoda users**: To log in, navigate to [http://localhost:3000](http://localhost:3000), where Grafana is running locally.
```

Comment:
I would perhaps change this to OSS and Sandbox users. You could completely remove mentioning the sandbox entirely here and just let the user know to get to Grafana on OSS go to this URL since we automatically handle the URL via the transformer

Reply:
absolutely. will change that and let ya know 😃